openapi-common/src/main/java/io/micronaut/openapi/swagger/core/util/PrimitiveType.java

```diff
@@ -505,6 +622,12 @@ public String getCommonName() {
 
     public abstract Schema<?> createProperty();
 
+    public abstract Schema<?> createProperty31();
```

Comment:
why is this method named `createProperty31`? I assume this is for OpenAPI 3.1? I don't think we should support multiple versions. As far as I see there is not an OpenAPI 3.2 finalized specification. I don't think we should move to OpenAPI 3.2. until that it is ready. https://github.com/OAI/OpenAPI-Specification/tags

Reply:
This class is copy of the same class from `swagger-core` with some my improvements. THis method from swagger `2.2.30`.

And we already support OpenAPI 3.1

Reply:
I know we support Open API 3.1. What is this PR solving? There are not tests in this PR.

Reply:
That PR not solve anything, just more coreect integration with swagger 2.2.30